tests/test_version.py

```diff
@@ -2,12 +2,14 @@
 
 
 def test_get_ucx_version():
-    ucp.reset()
+    # this ucp.reset() causes a segfault
+    # not sure whys
+    # ucp.reset()
```

Comment:
@madsbk this odd behavior.  When this particular `ucp.reset()` is uncommented I see a segfault.  Additionally, when I comment out all of `test_ucx_getters` and leave in the `ucp.reset()` in `test_version.py` I don't see any segfault

Reply:
When looking more closely at this code, I saw a potential problem and filed it as issue ( https://github.com/rapidsai/ucx-py/issues/292 ). It may not be the cause and I could have missed something, but figured it warranted mentioning.

Reply:
Thanks @jakirkham .  That's a good lead.  @madsbk  @jakirkham What do you think about merging this PR and come back to fixing this issue as the CI is currently blocking all other PRs

Reply:
Thanks @quasiben, the bug should be fixed in https://github.com/rapidsai/ucx-py/pull/289

Reply:
Ah, ok.  Great!